docs/intro/glossary.md

```diff
@@ -0,0 +1,100 @@
+
+- **Arbitrum Classic**: [Old Arbitrum stack](https://github.com/OffchainLabs/arbitrum) that used custom virtual machine ("AVM"); no public Arbitrum chain uses the classic stack as of 8/31/2022 (they instead use Nitro).
+
+- **Nitro**: Current Arbitrum tech stack; runs a fork of Geth directly on L2 and uses WebAssembly as its underlying VM for fraud proofs.
+
+- **Data Availability Committee (DAC)**: Permissioned set of parties responsibly for data availability in an AnyTrust chain.
+
+- **Data Availability Certificate**: Signed promise from a DAC of relevant data's availability as of a given Inbox hash. 
+
+- **Chain state**: A particular point in the history of an Arbitrum Chain. A chain state corresponds to a sequence of assertions that have been made, and a verdict about whether each of those assertions was accepted.
+
+- **Client**: A program running on a user's machine, often in the user's browser, that interacts with contracts on an Arbitrum chain and provides a user interface.
+
+- **Rollup Protocol**: Protocol for tracking the tree of assertions in an Arbitrum chain and their confirmation status.
+
+- **Speed Limit**: Target L2 computation limit for an Arbitrum chain.
+
+## Proving Fraud
+
+- **RBlock**: An assertion by an Arbitrum validator that represents a claim about an Arbitrum chain's state.
+
+- **L2 Block**: Data structure that represents a group of L2 transactions (analogous to L1 blocks).
+
+- **Challenge**: When two stakers disagree about the correct verdict on an assertion, those stakers can be put in a challenge. The challenge is refereed by the contracts on L1. Eventually one staker wins the challenge. The loser forfeits their stake. Half of the loser's stake is given to the winner, and the other half is burned.
+
+- **Confirmation**: The final decision by an Arbitrum chain to accept an RBlock as being a settled part of the chain's history. Once an assertion is confirmed, any L2 to L1 messages (i.e., withdrawals) can be executed.
+
+- **Challenge Period**: Window of time (1 week on Arbitrum One) over which an asserted RBlock can be challenged, and after which the RBlock can be confirmed.
+
+- **Dissection**: Process by which two challenging parties interactively narrow down their disagreement to a single computational step.
+
+- **One Step Proof**: Final step in a challenge; a single operation of the L2 VM (Wasm) is executed on L1, and the validity of its state transition is verified.
+
+
+- **Staker**: A party who deposits a stake, in ETH, to vouch for a particular RBlock in an Arbitrum Chain. A party who stakes on a false RBlock can expect to lose their stake. An honest staker can recover their stake once the node they are staked on has been confirmed.
+
+
+- **Active Validator**: A party who makes staked, disputable assertions about the state of the Arbitrum chain; i.e., proposing state updates or challenging the validity of assertions. (Not to be confused with the Sequencer)
```

Comment:
Don't know if you want to include this or not...
`Stake Latest`: stay staked on the latest node, challenging bad assertions
`Make Nodes`: Continually creates new nodes, challenging bad assertions

Reply:
gonna leave this out, as a bit too niche